cargo-crev/src/opts.rs

```diff
@@ -118,9 +118,6 @@ impl CargoOpts {
         if self.dev_dependencies && self.no_dev_dependencies {
             bail!("`--no-dev-dependencies` and `--dev-dependencies` can't be used together");
         }
-        if self.no_dev_dependencies {
-            writeln!(std::io::stderr(), "`--no-dev-dependencies` is the default, is now ignored and will be removed in the future")?;
-        }
```

Comment:
These lines should have been left in, right? `--no-dev-dependencies` is still the default and the option can be removed in the future. Or we should remove the option already.